### PR TITLE
feat: hide sf temporarily

### PR DESCRIPTION
**Pull request by tcheee**

### CODE REVIEW

**Review by oktapodia — APPROVED** [review]
LGTM

File: src/components/PartnerThemeFooterImage.tsx [hunk 1]
```diff
@@ -35,9 +35,7 @@ export const PartnerThemeFooterImage = () => {
         <BackgroundFooterImage
           alt="footer-image"
           src={
-            !!hasTheme && availableWidgetThemeMode === 'dark'
-              ? 'https://strapi.li.finance/uploads/superfest_light_sponsor_card_f92597365f.svg'
-              : 'https://strapi.li.finance/uploads/Superfest_sponsor_card_f3996bea6c.svg'
+            'https://strapi.li.finance/uploads/Superfest_sponsor_card_f3996bea6c.svg'
```
Comment (dennyscode) [inline src/components/PartnerThemeFooterImage.tsx:38]: Let us re-upload the footer-image before merging, as this link might change after.

File: src/components/Widgets/useWidgetTheme.ts [hunk 2]
```diff
@@ -42,7 +42,8 @@ export const useWidgetTheme = (): PartnerTheme => {
           },
           secondary: {
             // FIXME: we need to find out how to use the correct color from the main theme config
-            main: darkTheme.palette.accent2.main,
+            // main: darkTheme.palette.accent2.main,
+            main: '#ff0420',
```
Comment (dennyscode) [inline src/components/Widgets/useWidgetTheme.ts:46]: You were right about those colors which are used from the Theme passed into ThemeProvider, let´s keep theme.palette.secondary.main here?

File: src/providers/ThemeProvider.tsx [hunk 3]
```diff
@@ -150,7 +150,7 @@ export const ThemeProvider: React.FC<
             main: '#ff0420',
           },
           secondary: {
-            main: '#FDFBEF',
+            main: '#ff0420',
```
Comment (dennyscode) [inline src/providers/ThemeProvider.tsx:153]: --> Because you apply the red color here.

**Review by dennyscode — APPROVED** [review]

**Pull request merged**